README.md

```diff
@@ -12,17 +12,16 @@
 **TensorFlow** is an open source software library for numerical computation using
 data flow graphs.  The graph nodes represent mathematical operations, while
 the graph edges represent the multidimensional data arrays (tensors) that flow
-between them.  This flexible architecture enables you to deploy computation to one
-or more CPUs or GPUs in a desktop, server, or mobile device without rewriting
+between them.  This flexible architecture enables computation to CPUs or GPUs in a desktop, server, or mobile device without rewriting
```

Comment:
It's a bit subjective, but I think it's better to be less formal in an intro document. I'd undo this.